website/src/utils/tools.ts

```diff
@@ -0,0 +1,7 @@
+export function bytesToSize(bytes: number) {
```

Comment:
Use https://www.npmjs.com/package/pretty-bytes or another package is better?

Reply:
ok, This library is good

Reply:
wait a moment, this library inaccuracy